services/order.py

```diff
@@ -0,0 +1,56 @@
+from django.core.exceptions import ValidationError, ObjectDoesNotExist
+from django.db import transaction
+from django.utils.datetime_safe import datetime
+
+from db.models import Order, User, Ticket

```

Comment:
It's best practice to use `get_user_model()` instead of directly referencing `User`. This ensures compatibility with custom user models.